Sources/SmokeDynamoDB/InMemoryDynamoDBCompositePrimaryKeyTableStore.swift

```diff
@@ -167,49 +167,67 @@ internal class InMemoryDynamoDBCompositePrimaryKeyTableStore {
         return EventLoopFuture.andAllSucceed(futures, on: eventLoop)
     }
     
-    
     public func monomorphicBulkWriteWithoutThrowing<AttributesType, ItemType>(
         _ entries: [WriteEntry<AttributesType, ItemType>],
         eventLoop: EventLoop) async throws -> [Int: BatchStatementError] {
             var errors: [Int: BatchStatementError] = [:]
             
-            let futures = entries.enumerated().map { (index, entry) -> EventLoopFuture<Void> in
+            let futures = entries.enumerated().map { (index, entry) -> EventLoopFuture<Int> in
```

Comment:
It is probably a little cleaner to return EventLoopFuture<Int?> and use nil instead of -1 as the placeholder for success.

Reply:
yes, updated